packages/utils/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@graphql-tools/utils",
   "version": "7.2.6",
-  "description": "Common package containting utils and types for GraphQL tools",
+  "description": "Common package containing utils and types for GraphQL tools",
```

Comment:
This is a user visible change and presumably would require changing the version line above. But I'm leaving that alone for the time being.